src/cata_utility.h

```diff
@@ -40,7 +40,9 @@ double logarithmic( double t );
 double logarithmic_range( int min, int max, int pos );
 
 int bound_mod_to_vals( int val, int mod, int max, int min );
+std::string velocity_units( bool to_alternative_units = false );
```

Comment:
> Should I try to change return value to  char  or something else or it's okay to keep everything as is?

Difficult question and very subjective. Considering that it will (most likely) always return a string literal and never a dynamically created string, a `const char*` seems fine.

If you make it return a `const char*`, `weight_units` should probably be changed as well to make this consistent.